pkg/urbit/noun/events.c

```diff
@@ -782,96 +922,49 @@ _ce_image_fine(u3e_image* img_u,
 #endif
 
 /* _ce_image_copy():
-*/
+ */
 static c3_o
 _ce_image_copy(u3e_image* fom_u, u3e_image* tou_u)
 {
   c3_w i_w;
 
-  //  resize images
+  // resize images
   //
-  _ce_image_resize(tou_u, fom_u->pgs_w);
+  _ce_image_resize(tou_u, fom_u->pgs_w, NULL);
 
-  //  seek to begining of patch and images
+  // seek to begining of patch and images
   //
-  if (  (-1 == lseek(fom_u->fid_i, 0, SEEK_SET))
-     || (-1 == lseek(tou_u->fid_i, 0, SEEK_SET)) )
+  if ( (-1 == lseek(fom_u->fid_i, 0, SEEK_SET))
+       || (-1 == lseek(tou_u->fid_i, 0, SEEK_SET)) )
```

Comment:
> Incrementally restyling existing code while also changing functionality makes it harder to diff, review, or trace the history.

I agree, and at the same time, improving code hygiene is an incremental process. Our code style is inconsistent, particularly around whitespace, which is why I proposed the introduction of a linter in https://github.com/urbit/urbit/pull/5481. Because I met with resistance to linting, I've slipped into the habit of applying a more consistent style in whatever file I'm working in. I acknowledge that style changes intermingled with functionality is a pain, but so is a lack of a consistent coding style.

I've backed out the intrusive style changes, particularly in the `events` module. That said, you'll notice that I didn't revert the documentation changes in the `events` or `manage` modules because those changes provide more explicit explanation of what the code is doing and/or presents a more organized method of documenting. I also have refrained from backing out the replacement of bitwise page arithmetic with general arithmetic in `events` because I've now made numerous substantial changes to the `events` module, each of which has forced me to spend more time thinking about the math than I'd like to, which I attribute largely to the use of bitwise arithmetic. Furthermore, any compiler worth its salt will replace multiplication/division with a bit shift whenever possible, so using general arithmetic gets us the best of both worlds: the improved readability of general arithmetic and improved performance of bitwise arithmetic.